pkg/tfbridge/provider_test.go

```diff
@@ -248,13 +248,14 @@ func testIgnoreChanges(t *testing.T, provider *Provider) {
 	})
 	assert.NoError(t, err)
 
-	outs, err := plugin.UnmarshalProperties(createResp.GetProperties(), plugin.MarshalOptions{KeepUnknowns: true})
+	outs, err := plugin.UnmarshalProperties(createResp.GetProperties(), plugin.MarshalOptions{
+		KeepUnknowns: true,
+		SkipNulls:    true,
+	})
 	assert.NoError(t, err)
-	assert.True(t, resource.PropertyMap{
-		"id":                  resource.NewStringProperty(""),
```

Comment:
The PlanResourceChange version returns an unknown for "id", but that seems like may be OK.